modules.d/90livenet/livenetroot.sh

```diff
@@ -6,21 +6,33 @@ type getarg >/dev/null 2>&1 || . /lib/dracut-lib.sh
 . /lib/url-lib.sh
 
 PATH=/usr/sbin:/usr/bin:/sbin:/bin
+RETRIES={RETRIES:-100}
+SLEEP={SLEEP:-5}
 
 [ -e /tmp/livenet.downloaded ] && exit 0
 
 # args get passed from 40network/netroot
 netroot="$2"
 liveurl="${netroot#livenet:}"
 info "fetching $liveurl"
-imgfile=$(fetch_url "$liveurl")
 
-if [ $? != 0 ]; then
-	warn "failed to download live image: error $?"
-	exit 1
-fi
-
-> /tmp/livenet.downloaded
+imgfile=
+#retry until the imgfile is populated with data or the max retries
+for (( i=1; i<=$RETRIES; i++))
+do
+    imgfile=$(fetch_url "$liveurl")
+
+    if [ $? != 0 ]; then
+            warn "failed to download live image: error $?"
```

Comment:
Should we `exit 1` like [L20](https://github.com/dracutdevs/dracut/pull/467/files#diff-7088b30306c7d75f033518ea411e591cL20), when we've exceeded `$RETRIES`?